apps/docs/src/pages/components/code-editor.mdx

```diff
@@ -0,0 +1,243 @@
+import { Callout } from "nextra/components";
+import {
+  codeEditorClasses,
+  HvCodeEditor,
+} from "@hitachivantara/uikit-react-code-editor";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData(
+    "CodeEditor",
+    "code-editor",
+    codeEditorClasses,
+  );
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+To use the Code Editor component you'll have to install it from the `@hitachivantara/uikit-react-code-editor` package:
+
+```
+npm install @hitachivantara/uikit-react-code-editor
+```
+
+To use it in your app, import it from the `"@hitachivantara/uikit-react-code-editor` package and it's ready to use:
+
+```tsx live
+<HvCodeEditor />
+```
+
+You can specify a language throught the `language` prop to get proper syntax highlighting and define a default value:
+
+```tsx live
+<HvCodeEditor
+  height={270}
+  language="yaml"
+  value={`
+affinity: {}
```

Comment:
according to the doc block above, this should be `defaultValue`, otherwise it should be controlled